source/wasm/src/project_system.rs

```diff
@@ -29,6 +29,7 @@ export interface IProgramConfig {
     packageGraphSources: IPackageGraphSources;
     profile: TargetProfile;
     projectType: ProjectType;
+    isSingleFile: boolean;
```

Comment:
As I commented elsewhere (for the corresponding native struct) I think `isSingleFile` simply belongs in `PackageGraphSources` and the rest of the changes should fall into place

Reply:
done